cypress/integration/home_page_spec.js

```diff
@@ -43,23 +42,26 @@ describe('Username already exists', function() {
         cy.get('input').type(username);
         cy.get('button').click();
         cy.get('h3').should('contain', 'Username already exists');
-    })
-})
-// TO BE COMPLETED IN A TIC
-// describe('Game already started', function() {
-//     afterEach(() => {
-//         cy.request({
-//             url: '/api/end',
-//             method: 'GET'
-//         });
-//         cy.visit('/');
-//         });
-//     it('returns error', function () { 
-//         const username = 'Jane'
-//         cy.request('POST', '/auth/login', { username: 'Blane' });
-//         cy.
-//         cy.get('input').type(username);
-//         cy.get('button').click();
-//         cy.get('h3').should('contain', 'Game has already started');
-//     });
-// });
\ No newline at end of file
+    });
+});
+describe('Game already started', function() {
+    afterEach(() => {
```

Comment:
All of this `afterEach` and `beforeEach` stuff is redundant now, have a look at the tests that were just merged with develop to see how to change it